Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Description: use hotplug mode ("pinned spans") by default

Disable automatic span assignment. Span assignment should be done by
userspace udev hooks.

--- a/drivers/dahdi/dahdi-base.c
+++ b/drivers/dahdi/dahdi-base.c
@@ -7369,7 +7369,7 @@ int dahdi_assign_device_spans(struct dah
 	return 0;
 }
 
-static int auto_assign_spans = 1;
+static int auto_assign_spans = 0;
 static const char *UNKNOWN = "";
 
 /**
